### ISSUE TITLE
MNIST is using MSE loss

### ISSUE THREAD

**Issue by oxinabox** [FluxML/model-zoo#8]
MNIST is a classification problem,
it has a softmax output layer.
It should be using cross entropy loss.

**Comment by MikeInnes** [comment]
This is blocked on https://github.com/FluxML/Flux.jl/pull/98.

**Comment by MikeInnes** [comment]
Fixed in https://github.com/FluxML/model-zoo/commit/1b6e8723a2bb3d13423a1c8996823354acca3941
